pom.xml

```diff
@@ -23,7 +23,7 @@
   </licenses>
   <properties>
     <changelist>999999-SNAPSHOT</changelist>
-    <jenkins.version>2.440.1</jenkins.version>
+    <jenkins.version>2.440.3</jenkins.version>
```

Comment:
Why was this closed @alecharp? This change is also needed to match https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/